### PR TITLE
nsc-events-nextjs_7_432_role-authorized-archived-events

### PR DESCRIPTION
**Pull request by kaimanasse15**
Role Authorized update

### CODE REVIEW

**Review by taylorpapke — APPROVED** [review]
Looks like this is working for the archived-events and my-events page:
Screenshot 2024-06-04 at 10 51 37 PM

I can still access create-event page as a user. You will need to also fix the conflicts on the my-events page in order to pass CI.

File: app/create-event/page.tsx [hunk 1]
```diff
@@ -12,6 +12,9 @@ import { textFieldStyle } from "@/components/InputFields"
 import { MouseEvent, ChangeEvent, useState, FormEvent } from "react";
 import useAuth from "@/hooks/useAuth";
 
+import UnauthorizedPages from "@/components/UnauthorizedPages";
```
Comment (taylorpapke) [inline app/create-event/page.tsx:15]: Line 15 imports the UnauthorizedPages component but it is never used.

**Review by heosman — APPROVED** [review]
The archived events page and my events page show a 401 unauthorized error when not logged in or when logged in as a user. Like @taylorpapke mentioned, the create event page is still accessible to non-admin or non-creator users.
image

**Comment by heosman** [comment]
@kaimanasse15 @taylorpapke I added role authorization to the Create Event page properly, in addition to catching this branch up to main and resolving branch conflicts.

@SeattleColleges/leadership I'm unfamiliar with CI/CD so I'm unsure on how to resolve the Socket Security Alert.

**Pull request closed without merging**